docs/docs/spec.md

```diff
@@ -241,6 +244,11 @@ exceptions in the subsequent sections:
 
 -   an array component type
 
+    > An array of nullable strings is written `@Nullable String[]`. A variadic
+    > parameter whose type is "array of nullable strings" is written `@Nullable
+    > String...`. Compare this to the syntax for a nullable array of strings,
+    > shown above in the discussion of variadic parameters.
```

Comment:
It's too bad that we're putting variadic parameters first, and that we have to split these up at all.

Could the variadic note above simply point to this paragraph, and put all the examples here?

Reply:
Yes, that sounds better. Let me know what you think of this attempt.